app/controllers/EventsController.scala

```diff
@@ -39,4 +41,13 @@ trait AssessmentEventsController extends BaseController {
       assessmentEventsRepository.save(events)
     }.map(_ => Created).recover { case _ => UnprocessableEntity }
   }
+
+  def fetchEvents(eventTypeParam: String, venueParam: String) = Action.async { implicit request =>
```

Comment:
Minor point - We know eventTypeParam and venueParam are parameters. No need for the 'Param' part of the name.